runners/direct-java/src/main/java/org/apache/beam/runners/direct/DirectOptions.java

```diff
@@ -74,4 +76,10 @@ public Integer create(PipelineOptions options) {
       return Math.max(Runtime.getRuntime().availableProcessors(), MIN_PARALLELISM);
     }
   }
+
+  @Experimental(Kind.CORE_RUNNERS_ONLY)
+  @Default.Boolean(false)
```

Comment:
If the default is False, why is it causing performance degradation?

Reply:
(This is the revert; HEAD doesn't contain this flag, it always does the round trip)